md-dao/src/main/java/com/wipro/ats/bdre/md/dao/HistoryDataImportDAO.java

```diff
@@ -561,7 +555,7 @@
                     Intermediate rawTableColumns = (Intermediate) rawTableColumnsCriteria.list().get(0);
                     LOGGER.info("intermediate column : uuid " + rawTableColumns.getId().getUuid() + " key " + rawTableColumns.getId().getInterKey() + " value " + rawTableColumns.getInterValue());
                     String[] rawTableColumn = rawTableColumns.getInterValue().split(",");
-                    LOGGER.info("inter value after splitting " + rawTableColumn.toString());
+                    LOGGER.info("inter value after splitting " + Arrays.toString(rawTableColumn));
                     for ( int columnCounter = 1;columnCounter <= rawTableColumn.length; columnCounter ++){
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS134)